frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -87,6 +91,14 @@ export const OperatorHubItemDetails: React.SFC<OperatorHubItemDetailsProps> = ({
       item.subscription.metadata.name
     }?showDelete=true`;
 
+  const ibmMarketplaceLink = _.get(obj, [
```

Comment:
Calling this variable `ibmMarketplaceLink` is misleading since most of the time it's just the normal subscribe link.

Reply:
Is `createLinkWorkflow` better? Let me know if you want something else.